src/_posts/platform/databases/2000-01-01-maintenance-windows.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Database Maintenance Windows
+nav: Maintenance Windows
+modified_at: 2023-08-31 00:00:00
+tags: databases maintenance
+index: 0
+---
+
+Database Maintenance Windows is our system tailored to schedule the maintenances needed on your databases. It allows you to keep them secure and up-to-date, while being aware of the time period where your database could be temporarily unavailable.
+
+Each database has a default maintenance window set by Scalingo. But you can now provide a eight hours time window where Scalingo can proceed to the maintenance of your database.
+
+Moreover, it's also possible to schedule the maintenance when your app is less prone to high traffic (i.e : the night between Sunday and Monday). With those features, we have in mind the prevention of any undesired downtime of your app.
```

Comment:
It is not the case for everyone, I'd provide an example instead
```suggestion
Moreover, it's also possible to schedule the maintenance when your app is less prone to high traffic (e.g : the night between Sunday and Monday). With those features, we have in mind the prevention of any undesired downtime of your app.
```

Reply:
Nice catch! Fixing